src/components/Counter.test.js

```diff
@@ -0,0 +1,59 @@
+import { render, screen } from '@testing-library/react';
+import '@testing-library/jest-dom'
+import Counter from "./Counter";
+import { act } from 'react-dom/test-utils';
+
+describe("Check Counter element behavior", () => {
+
+    test("Test that component renders initial value provided in props", () => {
+
+        const propValue = 2;
+
+        render(<Counter defaultValue={propValue}/>)
+
+        const propValueElement = screen.getByText(/count/i);
+
+        expect(propValueElement).toBeInTheDocument();
+    })
+
+    test("Check if decrement button decrements the displayed value", async () => {
+        render(<Counter/>)
+
+        const countElement = screen.getByText(/count/i);
+
+        const countValue = countElement.textContent;
+
+        expect(countValue).toBe('Count: 0');
```

Comment:
Note: More convenient and clearer will be to contain the dynamic value in separated `</span>` with testId etc., and check it only.